### ISSUE TITLE
Evaluation Metrics

### ISSUE THREAD

**Issue by wh0330** [agakshat/visualdialog-pytorch#3]
Hi, thanks for your sharing. 
But, I'm a little confused. Do you evaluate with only 20 candidate answers?

**Comment by agakshat** [comment]
Hi @wh0330 

No, for the quantitative metrics we used the entire validation set of Visual Dialog v0.9 dataset (~40k images), while for the quantitative human evaluation we used ~100 samples. Any part of the code using only 20 answers for evaluation is probably a product of last-minute debugging being done.
